### ISSUE TITLE
Log In page:

### ISSUE THREAD

**Issue by LucieGueuning** [MSFREACH/MSF-REACH-COLLABORATION#134]
- [x] should stand in one page, no scroll down
- [x] quick links should be the same font/same size than the rest of the text
- [x] About MSF REACH should be a pop-up with shadow and horizontal and within one page (no scroll down)

**Comment by LucieGueuning** [comment]
(1) - (2) 
![image](https://user-images.githubusercontent.com/25951555/72154196-ab59a780-33b8-11ea-8bde-7a30347a6145.png)

(3) ![image](https://user-images.githubusercontent.com/25951555/72154216-b7de0000-33b8-11ea-946d-bb2d5d4f2023.png)

**Comment by hohangleung** [comment]
Updated, we also temporary added a label to display actual window size at the bottom-left of the page.
If the issue still exist, please provide the actual window size for reference.

**Comment by LucieGueuning** [comment]
Great. Thanks 

(1) Please make the white box close at the end. Follow the sketches 
![image](https://user-images.githubusercontent.com/25951555/72243821-8aca6100-35f5-11ea-97f8-a15f54e63619.png)

Versus
![image](https://user-images.githubusercontent.com/25951555/72243848-9c136d80-35f5-11ea-8a02-9d57237697f1.png)

(2) All the fonts should follow the sketches (e.g. Please sign in)

(3) the label - great. Can you make the same font and size than 'About MSF REACH'?

**Comment by hohangleung** [comment]
The white box margin is now adjusted based on 1422x680 window size. Please see if it's OK.

**Comment by LucieGueuning** [comment]
Please also implement: 
(2) All the fonts should follow the sketches (e.g. Please sign in)

(3) the label - great. Can you make the same font and size than 'About MSF REACH'?

**Comment by chiwaileung** [comment]
> Please also implement:
> (2) All the fonts should follow the sketches (e.g. Please sign in)
> 
> (3) the label - great. Can you make the same font and size than 'About MSF REACH'?

@hohangleung

**Comment by chiwaileung** [comment]
@LucieGueuning Please refer to #180 , please agree it and close this item, thanks!

**Comment by LucieGueuning** [comment]
I agree it is referring to 180. BUT please review this one specifically and place it to the next column 'RocTec Reviewed - to be review by Lucie' for easy tracking (even if it is duplication) if you confirm it is completed @chiwaileung
